rasa/nlu/featurizers/dense_featurizer/lm_featurizer.py

```diff
@@ -171,17 +211,12 @@ def cache_key(
         weights = component_meta.get("model_weights", {})
 
         return (
-            f"{cls.name}-{component_meta.get('model_name')}-"
+            f"{cls.__name__}-{component_meta.get('model_name')}-"
```

Comment:
Think we can remove the `cache_key` method since all the fingerprinting / caching is now done by the graph